.github/workflows/plugins_react_tests.yml

```
@@ -13,48 +13,60 @@ permissions:
   contents: read
 
 jobs:
+  setup_matrix:
+    name: Setup matrix
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.build_matrix.outputs.matrix }}
+    steps:
+      - name: Build test matrix
+        id: build_matrix
+        uses: ekohl/gha-matrix-builder@v0
```

Oh yes, and it was transferred
```suggestion
        uses: theforeman/gha-matrix-builder@v0
```